src/OnlineSales/Program.cs

```diff
@@ -434,4 +450,58 @@ private static void ConfigureCORS(WebApplicationBuilder builder)
             });
         });
     }
+
+    private static void ConfigureIdentity(WebApplicationBuilder builder)
+    {
+        builder.Services.AddIdentity<User, IdentityRole>()
+            .AddEntityFrameworkStores<PgDbContext>();
+        builder.Services.ConfigureApplicationCookie(options =>
+        {
+            // Cookie settings
+            options.Cookie.HttpOnly = true;
+            options.ExpireTimeSpan = TimeSpan.FromHours(12);
+
+            options.LoginPath = "/api/Identity/ExternalLogin";
```

Comment:
@Exmirai following our naming convention it should be /api/identity/external-login, must also be refactored from externallogin to external-login in swagger UI